log/namespaced/namespaced_test.go

```diff
@@ -0,0 +1,65 @@
+package namespaced
+
+import (
+	"testing"
+
+	"github.com/TheThingsNetwork/go-utils/handlers/entries"
+	apx "github.com/TheThingsNetwork/go-utils/log/apex"
+	apex "github.com/apex/log"
+	. "github.com/smartystreets/assertions"
```

Comment:
I would keep the package name:
`"github.com/TheThingsNetwork/go-utils/log/apex"` without alias 
and do `apex_log "github.com/apex/log"` avoid using to many aliases